### PR TITLE
update modal tab selection

### PR DESCRIPTION
**Pull request by jenngeorge**
- move tab selection js from script tags to scripts file
- highlight tab on hover and selection
- for the attractions and banquet modals, pre-select first tab
- move relevant styles to styles.scss and delete unused

### CODE REVIEW

File: sass/styles.scss [hunk 1]
```diff
@@ -247,6 +247,41 @@ Navigation
...
+
+  &:hover {
+    background: $mid-gray;
+    color: white;
+  }
+}
+
+// the selected tab in the tab bar
+.bar .selected {
+  background: $mid-gray !important;
```
Comment (jenngeorge) [inline sass/styles.scss:280]: Using !important isn't great. Might want to change hierarchy instead later.

**Comment by jenngeorge** [comment]
gonna rebase to pull in menu changes

File: sass/styles.scss (hunk 1 above)
Comment (gopherheiyuki) [reply sass/styles.scss:280]: It is c/p'ed from a W3 example that I don't fully understand. Feel free to make it follow best practices

**Review by gopherheiyuki — APPROVED** [review]

File: sass/styles.scss (hunk 1 above)
Comment (jenngeorge) [reply sass/styles.scss:280]: 👍 will address in a future commit!

**Pull request merged**